src/server/middlewares/redirects.ts

```diff
@@ -18,3 +20,21 @@ export const forceHost = ({
 
   await next()
 }
+
+export const startPageRedirect: IMiddleware<object> = async (ctx) => {
+  const actualIp = ctx.ip
+  const country = lookupCountry(actualIp)
+  const queryStringMaybe = ctx.querystring ? '?' + ctx.querystring : ''
+
+  if (!country || country === 'SE' || country === 'FI') {
```

Comment:
FI?

Reply:
The Finnish are best in the world at english in school AFAIK. I'm not sure if their Swedish is better than their English.

Reply:
I not know the praxis for redirecting Finnish people on Swedish websites however 🤔

Reply:
I have no clue tbh haha. It’s easy to switch language so maybe this is fine idk